core/opendaq/reader/include/opendaq/multi_reader.h

```diff
@@ -134,6 +134,15 @@ DECLARE_OPENDAQ_INTERFACE(IMultiReader, ISampleReader)
      */
     virtual ErrCode INTERFACE_FUNC getOffset(void* domainStart) = 0;
 
+    /*!
+     * @brief Gets the common sample rate in case input signal have different rates. The value of common sample rate is such
+     * that sample rate of any individual signal can be represented as commonSampleRate / Div, where Div is an integer. Unless
+     * the required common sample rate is specified in the MultiReader constructor, common sample rate is lowest common multiple
+     * of individual signal's sample rates. The number of samples to be read is specified in common sample rate.
+     * @param commonSampleRate The domain point at which the reader managed to synchronize all the signals.
+     */
+    virtual ErrCode INTERFACE_FUNC getCommonSampleRate(Int* commonSampleRate) = 0;
```

Comment:
Just a general question of what should we strive for @dejancrn @JakaMohorkoDS . Do we prioritize a nice API or a compatible API?
This sort-of unnecessarily breaks all users of Multi-Reader and forces people to update to new versions to avoid crashes and it could easily be avoided by adding functions at the end of the interface.

Its not just this PR but in general. I know the API is still unstable but if we can easily avoid breaking changes should we?

Reply:
I agree, maybe we should adopt this practice at the release of openDAQ 3.0